### DISCUSSION TITLE
Recommendation: Calculate the probability of being a mine on each cell using my packet

### DISCUSSION THREAD

**Discussion by eee555** [Bollos00/LibreMines#55] [Ideas]
https://github.com/eee555/ms_toollib

**Comment by eee555** [comment]
ms_toollib is a package for making minesweeper professionally, for rust, python, js, C++, and C language. With ms_toollib, you can do almost everything about developing minsweeper game, such as caculating probability, lay mine without guessing, .etc. here is a demo: 

[https://github.com/eee555/ms_toollib/blob/main/c%2B%2B/src/main.cpp](https://github.com/eee555/ms_toollib/blob/main/c%2B%2B/src/main.cpp)

**Comment by eee555** [comment]
It will save your time of writing the algorithm. You can contact me to develop more interfaces for you.

**Comment by Bollos00** [comment]
Hello,

Thanks for bring this to my attention. I will take a look.
